tools/clang/lib/SPIRV/SpirvEmitter.cpp

```diff
@@ -622,7 +623,7 @@ SpirvEmitter::getInterfacesForEntryPoint(SpirvFunction *entryPoint) {
   // declIdMapper keeps the mapping between variables with Input or Output
   // storage class and their storage class, we have to rely on
   // declIdMapper.collectStageVars() to collect them.
-  llvm::DenseSet<SpirvVariable *> interfaces;
+  llvm::SetVector<SpirvVariable *> interfaces;
```

Comment:
nit: do you think `SmallSetVector` would have benefits in terms of the memory usage?

Reply:
`SmallSetVector` is a "`SetVector` that performs no allocations if smaller than a certain size." but that has to be a const size (since it's part of the type). I wouldn't want to guess at what an appropriate number would be there to ensure it's actually an optimization without data to back it up.

I did switch to initializing with the `stageVars` iterator though, I missed that that was a constructor before.